client/src/views/users/UserDetailsModal.vue

```diff
@@ -0,0 +1,211 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <ms-modal
+      :title="$t('UsersPage.UserDetailsModal.title')"
+      :close-button="{ visible: true }"
+    >
+      <div>
+        <ion-text class="text1">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.name') }}
+        </ion-text>
+        <ion-text class="text1-right">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.joined') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text2">
+          {{ user.humanHandle.label }}
+        </ion-text>
+        <ion-text class="text2-right">
+          {{ timeSince(user.createdOn, '--', 'short') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text4">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.commonWorkspaces') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-chip
+          v-if="user.isRevoked()"
+          color="danger"
+        >
+          <ion-icon
+            :icon="ellipse"
+            class="revoked"
+          />
+          <ion-label>
+            {{ $t('UsersPage.UserDetailsModal.subtitles.revoked') }}
+          </ion-label>
+        </ion-chip>
+      </div>
+      <ion-list>
+        <ion-card
+          :disabled="user.isRevoked()"
+          class="workspace-card"
+        >
+          <ion-card-content>
+            <ion-avatar class="card-content-icons">
+              <ion-icon
+                class="card-content-icons__item"
+                :icon="business"
+              />
+              <ion-icon
+                class="cloud-overlay"
+                :class="'cloud-overlay-ko'"
+                :icon="cloudOffline"
+              />
+              <ion-text class="text3">
+                {{ $t('UsersPage.UserDetailsModal.placeholders.everyone') }}
```

Comment:
Don't add mocks values into translation files. There's a huge chance that we'll forget to remove them. It's perfectly fine to just write `{{ 'Everyone' }}` temporarily.

Or better yet, since you know this will be dynamically given by the bindings, even if you don't know how, you can already do a mock list straight into this component and iter over it:
```vue
<template>
  <ion-list>
    <ion-card
      v-for="workspace in workspaces"
    >
      {{ workspace.name }} {{ translateRole(workspace.role }} 
    </ion-card>
  </ion-list>
</template>

<script lang="ts">
interface MockSharedWorkspace {
  name: string;
  role: WorkspaceRole;
}

const workspaces = [{name: 'Workspace1', role: WorkspaceRole.Owner}, {name: 'Workspace2', role: WorkspaceRole.Contributor}];
</script>
```

It becomes very easy to swap the mock later on.